core/chains/evm/gas/block_history_estimator.go

```diff
@@ -747,7 +747,7 @@ func (b *BlockHistoryEstimator) getPricesFromBlocks(blocks []evmtypes.Block, eip
 			b.logger.Warnw(fmt.Sprintf("Block %v is not usable, %s", block.Number, err.Error()), "block", block, "err", err)
 		}
 		for _, tx := range block.Transactions {
-			if b.isUsable(tx, b.config, b.logger) {
+			if b.isUsable(tx, block, b.config, b.logger) {
```

Comment:
nit: Could we add Info Log when a Tx is deemed unusable? Provide the TxHash in the log message.
Or, perhaps just keep a count of unusable txes here, and for each block, put a log line saying x number of Txs in this block were unusable, out of totak n Txs. Also, atleast log 1 unusable TxHash in that log message, for debugging.

Reply:
Hmm, I just wanted to collect some summary of how many txs in a block were unusable, and perhaps, the reason.
But agree, not asking to do any extra logging or processing.